main/src/cgeo/geocaching/network/Network.java

```diff
@@ -52,6 +53,8 @@
 import okhttp3.RequestBody;
 import okhttp3.Response;
 import okhttp3.TlsVersion;
+import okhttp3.internal.Util;
+
```

Comment:
see https://ci.cgeo.org/job/cgeo%20pull%20request/2611/checkstyleResult/new/
remove extra separation in import group before 'org.apache.commons.compress.utils.IOUtils'